pkg/meta/tkv.go

```diff
@@ -3756,3 +3780,61 @@ func (m *kvMeta) loadDumpedACLs(ctx Context) error {
 		return nil
 	})
 }
+
+func (m *kvMeta) getDirFetcher(ctx Context) dirFetcher {
+	return func(inode Ino, cursor interface{}, offset, limit int, plus bool) (interface{}, []*Entry, error) {
+		var startKey []byte
+		total := limit
+		if cursor == nil {
+			startKey = m.entryKey(inode, "")
+
+			// should not happen
```

Comment:
Also believe this should not happen. Even the syscall `readdir/getdents64` can pass a directory offset. For most app process, reading a directory is typically performed sequentially.